mars/oscar/backends/ray/pool.py

```diff
@@ -107,7 +120,8 @@ async def _kill_actor_forcibly(self, process: 'ray.actor.ActorHandle'):
 
     async def is_sub_pool_alive(self, process: 'ray.actor.ActorHandle'):
         try:
-            await process.health_check.remote()
```

Comment:
If the `health_check` is no longer used, we should remove the method and the `PoolStatus`.